helix-term/src/commands.rs

```diff
@@ -3161,60 +3162,72 @@ pub mod insert {
         use helix_lsp::lsp;
         // if ch matches completion char, trigger completion
         let doc = doc_mut!(cx.editor);
-        let language_server = match doc.language_server() {
-            Some(language_server) => language_server,
-            None => return,
-        };
+        let trigger_completion = doc
```

Comment:
This will refresh all completions even if the trigger char just belong to one of the servers. I want to rework the way trigger chars work anyway so we can leave this for now but that should be adressed in the future

Reply:
Yes maybe not completely optimal, but I think it would be quite a little bit of rework to support just one or some of the language servers (filter by trigger character?), without creating weird behavior (e.g. one language server would support completion items at this position, even if not triggered, but only the triggered language server provides completion items).

So yeah I also think this should definitely be isolated in a separate PR to be implemented/discussed properly.